test/Dapr.Client.Test/TestData/InvocationHandlerTestsData.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Xunit;
```

Comment:
Please put these back with the inline data attributes. When building unit tests, I don't expect to have to look in a separate file to add additional variations, I expect to see them inline using the convenience attributes provided by the framework. This might make sense in an actual library, but not in a unit test class.